docs/content/management-bundles.md

```diff
@@ -252,6 +252,49 @@ and number keys are converted to strings. Also, YAML !!binary tags are not
 supported.
 {{< /info >}}
 
+### Bundle Container Format
+
+OPA is able to interact with OCI compatible registries to be able to download and use policies stored as containers.
+To configure OPA to use an OCI repository see the [service configuration section](https://www.openpolicyagent.org/docs/latest/configuration/#services)
+
+The [persistence directory setting](https://www.openpolicyagent.org/docs/latest/configuration/#miscellaneous) allows users to specify the location of the download policies. For OCI compatible registries an ***oci*** folder is created in this specified location with a local registry. As the OCI downloader plugin requires a storage path if the persistence directory is not set it will use the system's temporary folder location. Future work to ensure the management of the persistence directory is scoped for the OCI container support.
+
+**Current Limitations**
+The OCI Downloader plugin used by OPA has a couple of limitation:
+- it accepts only **one** layer per image that contains the bundle tarball
+- it can download only the following application media types: 
+    - `application/vnd.oci.image.layer.v1.tar+gzip`
+    - `application/vnd.oci.image.manifest.v1+json`
+    - `application/vnd.oci.image.config.v1+json`
+- cannot create/push image using the OPA CLI 
+- it can download only from OCI compatible registries 
+
+#### Building and Publishing Policy Containers
+
+There are multiple ways to build an image from a policy code base using different tools.
+
+##### Using policy CLI
+
+The [policy CLI](https://www.openpolicyregistry.io/docs/cli/download) tool can be easily used to build and push a policy bundle to a remote OCI registry using just two simple commands:
+- `policy build <path_to_src> -t <org>/<repo>:<tag>`
+- `policy push <registry>/<org>/<repo>:<tag>`
+A full tutorial is available [here](https://www.openpolicyregistry.io/docs/tutorial)
+
+##### Using OPA and ORAS CLIs
+
+To build and push a policy bundle to a remote OCI registry with the [OPA CLI](https://www.openpolicyagent.org/docs/edge/cli/) and [ORAS CLI](https://oras.land/cli/) you can  use the following commands:
+
+- `opa build <path_to_src>` will allow you to build a bundle tarball from your OPA policy and data files
+
+Now that we have the tarball we will need to provide a config manifest to the ORAS CLI and the tarball itself: 
+- `oras push <registry>/<org>/<repo>:<tag> --manifest-config <you_config_json>:application/vnd.oci.image.config.v1+json <the_tarball_obtained_from_opa_build>:application/vnd.oci.image.layer.v1.tar+gzip`
+
+Using an empty(`{}`) `manifest-config` json file should be sufficient to be able to push and allow the OCI downloader to use the remote policy image. 
+
```

Comment:
For a complete documentation experience,

- Let's also add an example how you'd configure OPA to download that policy stored with an OCI registry
- Can I pin a version/tag/ref? 
- When will OPA redownload and activate an updated OCI bundle?